packages/editor/src/components/editor-canvas/index.js

```diff
@@ -47,19 +47,25 @@ const interfaceLabels = {
 
 export default function EditorCanvas( {
 	className,
+	enableRegionNavigation,
 	styles,
 	children,
 	forceIsDirty,
 	contentRef,
 	disableIframe,
 	autoFocus,
+	customSaveButton,
+	forceDisableBlockTools,
+	title,
+	iframeProps,
```

Comment:
I wish we didn't need all these props to customize the behavior of the editor between post and site editors. But maybe we could iterate (remove some of these) in dedicated PRs.